app/models/role.rb

```diff
@@ -1,6 +1,7 @@
-class Role < ActiveRecord::Base
-  has_many :permissions
-  has_many :users, through: :permissions
+Role = Struct.new(:name, :applications) do
+
+  def <=>(other)
+    name <=> other.name
+  end
```

Comment:
Using Structs in production code not the best idea: http://thepugautomatic.com/2013/08/struct-inheritance-is-overused/
Convert to a PORO with an initialise method